app/src/main/java/cmput301/team19/lendz/AddBookFragment.java

```diff
@@ -0,0 +1,64 @@
+package cmput301.team19.lendz;
+
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link AddBookFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class AddBookFragment extends Fragment {
```

Comment:
We can just use the upcoming `EditBookFragment` instead.

Reply:
Thats sth we can clarify tomorrow